test/functional/attribute_test.rb

```diff
@@ -0,0 +1,654 @@
+require File.join(File.dirname(__FILE__), 'test_helper')
```

Comment:
This file should be renamed to `compute_attribute_test.rb` to match the module file it is testing.